lib/tpl/tabs_cmi_view.hamlc

```diff
@@ -1,3 +1,5 @@
 %cmi-tabs( no-slide )
   - @getTabs().forEach (tab) ->
-    %cmi-tab( data-model-id="#{tab.id}" )= tab.name
\ No newline at end of file
+    - if tab.badge
+      %paper-badge( for="#{tab.id}" label="#{tab.badge}" )
+    %cmi-tab( data-model-id="#{tab.id}", id="#{tab.id}" )= tab.name
```

Comment:
You might get a problem here. Think about what happens, if this vm is used twice on one website. 

A solution would be to pre-scope the id with the views `@cid `.

```coffeescript
templateHelpers: =>
  getViewCid: => @cid
```

```haml
%paper-badge( for="#{@getViewCid()}-#{tab.id}" label="#{tab.badge}" )
-# and
%cmi-tab( data-model-id="#{tab.id}", id="#{@getViewCid()}-#{tab.id}" )= tab.name
```